docs/content/preview/secure/enable-authentication/yedis.md

```diff
@@ -7,7 +7,7 @@ headcontent: Enable Users in YEDIS.
 image: /images/section_icons/secure/authentication.png
```

Comment:
title: Enable Users in YEDIS
headerTitle: Enable Users in YEDIS
linkTitle: Enable Users in YEDIS
description: Enable Users in YEDIS.
headcontent: Enable Users in YEDIS.

change to 
title: Enable users in YEDIS
headerTitle: Enable users in YEDIS
linkTitle: Enable users in YEDIS
description: Enable users in YEDIS.
headcontent: Enable users in YEDIS.